subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/maven/MavenVersionRangeResolveIntegrationTest.groovy

```diff
@@ -58,4 +58,43 @@ dependencies {
             }
         }
     }
+
+    @Issue("https://github.com/gradle/gradle/issues/1898")
+    def "can resolve parent pom with version range"() {
```

Comment:
I wonder if it's also possible to have a version range in the coordinates for an _imported_ POM?

Reply:
[Here](https://github.com/blindpirate/gradle/blob/464c412b272e0a64bd25d1d9fc0ab71144ec617f/subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/maven/MavenVersionRangeResolveIntegrationTest.groovy#L104) it is.